src/pages/home/ReportScreen.js

```diff
@@ -396,6 +397,7 @@ export default compose(
     withLocalize,
     withWindowDimensions,
     withNetwork(),
+    withCurrentReportID,
```

Comment:
I believe it would be appropriate to actually use `currentReportID` [here](https://github.com/Expensify/App/blob/c20c3870ee46d6788de554be27706e9898638904/src/pages/home/ReportScreen.js#L160)